discuss/email_servers.rst

```diff
@@ -35,6 +34,13 @@ Here is a typical configuration for a G Suite server.
 
 Then set your email domain name in the General Settings.
 
+.. note:: If you get a *[AUTHENTICATIONFAILED] Invalid credentials (Failure)* warning when you *Test
+   Connection* on a Gmail address, activate the *Less secure app access* option.
+   A direct link can be `accessed here. <https://myaccount.google.com/lesssecureapps?pli=1>`_
```

Comment:
Nitpicking: We generally avoid returning to a new line in the code if it does not results in a new line in the render. This is purely to avoid misleading writers and make them believe that a new line will be rendered.